### ISSUE TITLE
False positive: Unresolved reference

### ISSUE THREAD

**Issue by kryptan** [intellij-rust/intellij-rust#4731]
## Environment

* **IntelliJ Rust plugin version:** 0.2.111.2147-193

## Problem description
![image](https://user-images.githubusercontent.com/11528855/70350095-1b838380-1877-11ea-95d5-0a5203f84fb1.png)

## Steps to reproduce

```Rust
mod module {
    pub trait Trait {
        fn function() -> Self;
    }

    impl Trait for u8 {
        fn function() -> Self {
            0
        }
    }
}

fn main() {
    let _: u8 = module::Trait::function();
}
```

**Comment by Kubikx** [comment]
0.2.113.2150-193 still hapening for a lot of code :(

**Comment by dima74** [comment]
Looks like it was fixed some time ago. Feel free to reopen or create a new issue if you encounter any problems
